documents/src/pages/elements/chart.md

```diff
@@ -1502,9 +1502,9 @@ ChartJS.register(
 );
 ```
 
-## Localization
+## Localisation
 
-To get started, set `adapters.date` of `options.scales[scaleId]` with [date-fns](https://date-fns.org) locale.
+To localise date time in chart, you need a utility library, [date-fns](https://date-fns.org), and set the localiser to `adapters.date` in `options.scales[scaleId]`.
```

Comment:
```suggestion
To localise date time in chart, you can use a utility library, [date-fns](https://date-fns.org), and set the localiser to `adapters.date` in `options.scales[scaleId]`.
```

Reply:
Our package has date-fns using in datetime-picker, I want to lead a user that he has no installation required option.